### PR TITLE
Update action table

### PR DESCRIPTION
**Pull request by alekhyaku**
Action table with actual characters class corresponding actions

### CODE REVIEW

**Review by OddJerbb — APPROVED** [review]
👍

**Review by KellyPSun — APPROVED** [review]
The text for longer actions are cut off. I'm gonna approve so Jalyn can be unblocked since it's easy to fix later.

**Review by jcderry5 — CHANGES_REQUESTED** [review]

File: step-strikers-app/step-strikers-app/Battle Screens/BattleSelectActionViewController.swift [hunk 1]
```diff
@@ -174,16 +174,25 @@ class BattleSelectActionViewController: UIViewController, UITableViewDataSource,
     
     // TODO: update array with actual actions player can do
```
[the remark this answers is not in the document]
Comment (jcderry5) [reply step-strikers-app/step-strikers-app/Battle Screens/BattleSelectActionViewController.swift:175]: + the TODO @Jalyn regarding where what holds the action string that the player clicked on in the ViewController file

**Comment by alekhyaku** [comment]
Ok all changes done, also forgot to add on the commit but i think i removed the extraneous dice sprite as well

**Review by OddJerbb — APPROVED** [review]

**Pull request merged**